finat/piola_mapped.py

```diff
@@ -0,0 +1,84 @@
+import numpy
+from gem import Literal
+
+
+def determinant(A):
```

Comment:
This is about the worst possible algorithm for `n x n` determinants.  I assume the usage is for a small and probably symbolic matrix?  If so, should this (and the `adjugate` function be lifted into GEM someplace?

Reply:
Even in 3D, we only call this code for 2 x 2 matrices.